templates/ssl_ratios_v1.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-derived-datasets.telemetry.ssl_ratios_v1` AS
+WITH windowed AS (
+  SELECT
+    *,
+    SUM(ssl_loads) OVER w1 + SUM(non_ssl_loads) OVER w1 AS total_loads
+  FROM
+    `moz-fx-data-derived-datasets.telemetry.ssl_ratios_raw_v1`
+  WINDOW
+    w1 AS (ORDER BY submission_date ROWS BETWEEN UNBOUNDED PRECEDING AND UNBOUNDED FOLLOWING))
```

Comment:
@mreid-moz I think we should use a view with an intermediate table, like this, so that `total_loads` can be inexpensively calculated at query time, and be consistent with the existing meaning